litmus-portal/graphql-server/manifests/cluster-subscriber.yml

```diff
@@ -1083,6 +1083,13 @@ rules:
       - configmaps
     verbs:
       - get
+  - apiGroups:
+      - litmuschaos.io
+    resources:
+      - chaosengines
+      - chaosresults
+    verbs:
+      - get  
```

Comment:
is get sufficient? or do we need get & list

Reply:
No just get is working fine..